### DISCUSSION TITLE
Request-PnPReIndexWeb on SharePoint Site not working only on root site

### DISCUSSION THREAD

**Discussion by ZucciDevs** [pnp/powershell#3813] [General]
### Reporting an Issue or Missing Feature
Running Request-PnPReIndexWeb seems to only initiate when run on root site (https://yourSPTenant.sharepoint.com) but not on other sites that reside ( https://yourSPTenant.sharepoint.com/sites/*)

### Expected behavior 
Output should mark any site for full reindexing

### Actual behavior 
Returns an error that does not pertain to action requested:

Request-PnPReIndexWeb : You are not allowed to rename this library.
At line:1 char:1
+ Request-PnPReIndexWeb
+ ~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Request-PnPReIndexWeb], PSInvalidOperationException
    + FullyQualifiedErrorId : InvalidOperation,PnP.PowerShell.Commands.RequestReIndexWeb

### Steps to reproduce behavior
Connect-PnpOnline -Url "your tenant and site here" -Interactive
Request-PnPReIndexWeb

### What is the version of the Cmdlet module you are running?
1.12.0

### Which operating system/environment are you running PnP PowerShell on?
- [ X ] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by NoralK** [comment]
@ZucciDevs - This working for me in both v1.12.0 and v2.4.0 of PnP.Powershell.

I Tried `https://<<tenant>>.sharepoint.com` and `https://<<tenant>>.sharepoint.com/sites/<<somesite>>`.  If you are trying to use a wild-card `*` that will not work.

Could it be permission related?  I have the SharePoint Tenant Admin role and I am part of the Admins of the site collection.
